sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -247,22 +261,27 @@ export class InstitutionController extends BaseController {
     );
   }
 
+  @IsInstitutionAdmin()
   @Patch("user-status/:userName")
   async updateUserStatus(
+    @UserToken() token: IInstitutionUserToken,
     @Param("userName") userName: string,
     @Body() body: UserDto,
   ): Promise<void> {
     // Check  user exists or not
-    // TODO: Check if the user belongs to the institution.
-    // We can pass the institution id to getInstitutionUserByUserName to ensure it.
-    // The institution id is present on token.
     const institutionUser =
       await this.institutionService.getInstitutionUserByUserName(userName);
     if (!institutionUser) {
       throw new UnprocessableEntityException(
         "user does not exist in the system.",
       );
     }
+    // checking if user belong to logged-in users institution
```

Comment:
Now that we have the IsInstitutionAdmin added we can remove the lines 274-277.
Please consider removing the above-mentioned lines.